src/Shared/ProblemDetails/ProblemDetailsDefaults.cs

```diff
@@ -63,11 +76,35 @@ internal static class ProblemDetailsDefaults
             "Unprocessable Entity"
         ),
 
+        [426] =
+        (
+            "https://tools.ietf.org/html/rfc9110#section-15.5.22",
+            "Upgrade Required"
+        ),
+
```

Comment:
429? I think it'll be even more common with rate limiting built into the framework.

Reply:
Makes sense to add if the team is happy with the other additions I already made.

Reply:
I don't see why not add 429 as well.

Reply:
I remember why I didn't add it now - it's not in the RFC 😄